internal/bus/chainsubscriber.go

```diff
@@ -433,8 +435,14 @@ func (s *chainSubscriber) broadcastExpiredFileContractResolutions(tx sql.ChainUp
 
 		// verify txn and broadcast it
 		_, err = s.cm.AddV2PoolTransactions(basis, []types.V2Transaction{txn})
-		if err != nil {
+		if err != nil &&
+			(strings.Contains(err.Error(), "has already been resolved") ||
+				strings.Contains(err.Error(), "not present in the accumulator")) {
```

Comment:
I'd use `utils.IsErr` and define these two errors, this spot is important enough to warrant extracting these into error  variables